modules/oe_whitelabel_helper/src/Plugin/field_group/FieldGroupFormatter/MultiColumn.php

```diff
@@ -0,0 +1,266 @@
+
+  /**
+   * Constructs a Popup object.
+   *
+   * @param string $plugin_id
+   *   The plugin_id for the formatter.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param object $group
+   *   The group object.
+   * @param array $settings
+   *   The formatter settings.
+   * @param string $label
+   *   The formatter label.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler service.
+   * @param \Drupal\Core\Entity\EntityFieldManagerInterface $entity_field_manager
+   *   The Entity field manager.
+   */
+  public function __construct($plugin_id, $plugin_definition, $group, array $settings, $label, ModuleHandlerInterface $module_handler, EntityFieldManagerInterface $entity_field_manager) {
+    parent::__construct($plugin_id, $plugin_definition, $group, $settings, $label);
+    $this->moduleHandler = $module_handler;
+    $this->entityFieldManager = $entity_field_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $plugin_id,
+      $plugin_definition,
+      $configuration['group'],
+      $configuration['settings'],
+      $configuration['label'],
+      $container->get('module_handler'),
+      $container->get('entity_field.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultContextSettings($context): array {
+    return [
+      'first_column' => '',
+      'second_column' => '',
+      'hide_table_if_empty' => FALSE,
+    ] + parent::defaultSettings($context);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(): array {
+    $form = parent::settingsForm();
+
+    $form['first_column'] = [
+      '#title' => $this->t('First column classes'),
+      '#description' => $this->t('Add custom classes to the first column.'),
+      '#type' => 'textfield',
+      '#default_value' => $this->getSetting('first_column'),
+    ];
+    $form['second_column'] = [
+      '#title' => $this->t('Second column classes'),
+      '#description' => $this->t('Add custom classes to the second column.'),
+      '#type' => 'textfield',
+      '#default_value' => $this->getSetting('second_column'),
+    ];
+
+    $form['hide_table_if_empty'] = [
+      '#title' => $this->t('Hide the table if empty'),
+      '#description' => $this->t('Do not output any table or container markup if there are no rows with values.'),
+      '#type' => 'checkbox',
+      '#default_value' => $this->getSetting('hide_table_if_empty'),
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsSummary(): array {
+    $summary = parent::settingsSummary();
+    $summary[] = $this->t('Display results as a 2 column table.');
+
+    return $summary;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function preRender(&$element, $rendering_object): void {
```

Comment:
We have an issue with this according to the complexity, please have a look at https://drone.fpfis.eu/openeuropa/oe_whitelabel/483/20